### ISSUE TITLE
Dark mode too high-contrast

### ISSUE THREAD

**Issue by dbohdan** [vmiklos/plees-tracker#506]
This is what Plees Tracker 24.8 looks like on my phone in light and dark mode. I find light mode easier to look at in the dark than dark mode. I am sure this is because in dark mode the solid white text and especially "Sleeps" and "Daily" are too bright against the solid black (#000000) background.

<a href="https://github.com/user-attachments/assets/3495ce8e-5a74-4222-a504-b783aad59e36"><img alt="Light-mode screenshot." src="https://github.com/user-attachments/assets/3495ce8e-5a74-4222-a504-b783aad59e36" height="300"></a> <a href="https://github.com/user-attachments/assets/32474997-e722-42c3-a6cc-a40829309854"><img alt="Dark-mode screenshot." src="https://github.com/user-attachments/assets/32474997-e722-42c3-a6cc-a40829309854" height="300"></a>

It would be nice if the user interface had lower contrast in dark mode. For comparison, here is a "before" and "after" mockup made by Claude and slightly edited to change the colors. I don't mean that you should use these exact colors; this is just to illustrate the idea.

<a href="https://github.com/user-attachments/assets/f276e88e-395b-46d4-98b8-d4b57cb8faf1"><img alt="A mockup with the original colors." src="https://github.com/user-attachments/assets/f276e88e-395b-46d4-98b8-d4b57cb8faf1" height="300"> <a href="https://github.com/user-attachments/assets/0c12abcd-c2d8-448d-a727-49d69ee7fc20"><img alt="A mockup with updated colors." src="https://github.com/user-attachments/assets/0c12abcd-c2d8-448d-a727-49d69ee7fc20" height="300">

**Comment by vmiklos** [comment]
Yes, de literally dark background is indeed poor. Just checking, what you request is reducing contrast by changing the 1) background, 2) sleeps and 2) average of daily suns colors, correct?

**Comment by dbohdan** [comment]
My request is to have the contrast of the following UI elements reduced:

- The background (#000000)
- #FFFFFF text:
    - The headings "Plees Tracker", "Status", "Sleeping since&nbsp;[...]"/"Tracking stopped", "Dashboard (Last week)", and "Past sleeps"
    - The text of "Sleeps" and "Daily"
    - The text on the "Start"/"Stop" button?
- The background color of "Sleeps" and "Daily"

At a lower priority, I think the icons for each item could also be adjusted for contrast. I mean those:
<img alt="A crop of the dark-mode screenshot with the icons." src="https://github.com/user-attachments/assets/1923fa73-9186-4418-a36d-f47343c5d8f8" height="81">

**Comment by vmiklos** [comment]
Once the above is in, dark mode looks like this:

![image](https://github.com/user-attachments/assets/0d764f03-f8ef-4779-a524-3c9210f34238)

The icons are also doable, I think -- just need to came up with exact lower-contrast green/red/blue RGB values.

Do you build from source or is it useful if I cut a new release to trigger an fdroid update?

**Comment by dbohdan** [comment]
Thanks! The new screenshot looks like a great improvement. I have Plees Tracker installed from F-Droid, so a release would be good.

**Comment by vmiklos** [comment]
Ok, https://github.com/vmiklos/plees-tracker/releases/tag/v24.8.1 is on it's way to fdroid
